python/test/t_LHS_std.py

```diff
@@ -1,48 +0,0 @@
-#! /usr/bin/env python
-
-import openturns as ot
```

Comment:
Removing this unit test makes sense, because the class is deprecated. But it seems to me that the content may have some value to check that the suggested ProbabilitySimulationAlgorithm + LHSExperiment combo provide the same feature. I suggest to move the content to the unit test of the ProbabilitySimulationAlgorithm.

Reply:
its already tested with LHSExperiment

Reply:
Ok, that is written in [t_ProbabilitySimulationAlgorithm_std](https://github.com/jschueller/openturns/blob/c9cba6ad56e49da047417ca7c22a6ff4b751ff41/python/test/t_ProbabilitySimulationAlgorithm_std.py#L62). Ok for me.